docs/developer-guide/create-a-stream.rst

```diff
@@ -0,0 +1,265 @@
+.. _create-a-stream-with-ksql:
+
+Create a KSQL Stream
+====================
+
+In KSQL, you create streams from Kafka topics, and you create streams of
+query results from other streams.
+
+* Use the CREATE STREAM statement to create a stream from a Kafka topic.
+* Use the CREATE STREAM AS SELECT statement to create a query stream from an
+  existing stream.
+
+.. note::
+
+   Creating tables is similar to creating streams. For more information, see

```

Comment:
I'll update this link when I write the corresponding "Create a KSQL Table" topic.